bin/npm-explicit-installs.js

```diff
@@ -21,6 +24,42 @@ require('yargs')
     })
   })
   .command('delete', 'delete packages from the home page', function () {
+    var packages = require('../packages')
+    inquirer.prompt({
+      name: 'package',
+      message: 'remove package from homepage',
+      type: 'list',
+      choices: packages,
+    }, function (answer) {
+      packages.splice(packages.indexOf(answer.package), 1)
+      fs.writeFileSync(path.resolve(__dirname, '../packages.json'), JSON.stringify(packages, null, 2), 'utf-8')
+    })
+  })
+  .command('add', 'add a new package to the home page', function () {
+    var packages = require('../packages')
+    var logos = require('../logos')
+
+    inquirer.prompt([
+      {
+        name: 'package',
+        message: 'name of package to add',
+        validate: function (input) {
+          if (!input.length) return 'you must provide a package name'
+          else return true
+        }
+      },
+      {
+        name: 'logo',
+        message: 'url of icon to use for package (optional)'
+      }
+    ], function (answer) {
+      if (answer.logo) {
+        logos[answer.package] = answer.logo
+        fs.writeFileSync(path.resolve(__dirname, '../logos.json'), JSON.stringify(logos, null, 2), 'utf-8')
+      }
+      packages.push(answer.package)
+      fs.writeFileSync(path.resolve(__dirname, '../packages.json'), JSON.stringify(packages, null, 2), 'utf-8')
```

Comment:
Would be nice to specify order somehow. Maybe a separate `reorder` command? I guess reordering would have to explicitly bust the cache too since `_.xor` ignores ordering.